aks/postgres/outputs.tf

```diff
@@ -26,15 +26,16 @@ output "name" {
 }
 
 output "url" {
-  value     = "postgres://${urlencode(local.database_username)}:${urlencode(local.database_password)}@${local.host}:${local.port}/${local.database_name}"
+  value     = "postgres://${urlencode(local.database_username)}:${urlencode(local.database_password)}@${local.host}:${local.port}/${local.database_name}?sslmode=${var.use_azure ? "require" : "disable"}"
   sensitive = true
 }
 
 output "dotnet_connection_string" {
-  value     = "Server=${local.host};Database=${local.database_name};Port=${local.port};User Id=${local.database_username};Password='${local.database_password}';Ssl Mode=Require;Trust Server Certificate=true"
+  value     = "Server=${local.host};Database=${local.database_name};Port=${local.port};User Id=${local.database_username};Password='${local.database_password}';Ssl Mode=${var.use_azure ? "require" : "disable"};Trust Server Certificate=true"
```

Comment:
According to https://www.npgsql.org/doc/security.html?tabs=tabid-1 it should be Disable. Or Prefer, to always try SSL first.

Reply:
fixed